src/System.IO.FileSystem.AccessControl/tests/FileSystemAclExtensionsTests.cs

```diff
@@ -11,12 +11,14 @@ namespace System.IO
     public class FileSystemAclExtensionsTests
     {
         [Fact]
+        [SkipOnTargetFramework(TargetFrameworkMonikers.Uap | TargetFrameworkMonikers.UapAot)]
```

Comment:
Also, please include a reason message in the attribute.  It should link back to the associated issue.